EIPS/eip-4906.md

```diff
@@ -2,7 +2,7 @@
 eip: 4906
 title: ERC-721 Metadata Update Extension
 description: Add a `MetadataUpdate` event to ERC-721.
-author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>
+author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>, Nathan (@niftynathangang)
```

Comment:
Have you checked that this author change is okay?

Reply:
@0xanders - are you comfortable with me adding myself as an author?

Reply:
> คุณได้ตรวจสอบแล้วว่าการเปลี่ยนแปลงของผู้เขียนคนนี้โอเคไหม?

ขอบคุณค่ะ หนูไม่ค่อยเข้าใจเลยไม่สามารถเอาอะไรมาตัดสิน
แต่ขอให้ทำในสิ่งที่คิดว่าถูกต้องที่สุดนะคะ

Reply:
> ด้วยความยินดีค่ะ

Reply:
> @0xanders - are you comfortable with me adding myself as an author?

I suggest that  change `Nathan (@niftynathangang)` to `Nathan <your email>` .

Reply:
GitHub is generally preferred over email when possible, as it allows the bot to mention the person reliably.

Reply:
For the time being, I switched to my e-mail, per @0xanders

Reply:
I second MicahZoltu, I strongly recommend using your GitHub handle.

Reply:
Changed back to github handle